motivaa-backend/motivaa-app/src/test/java/backendtesting/CookieHandler.java

```diff
@@ -0,0 +1,60 @@
+package backendtesting;
+
+import io.restassured.RestAssured;
+import io.restassured.response.Response;
+import lombok.extern.log4j.Log4j2;
+
+import java.util.Arrays;
+
+@Log4j2
+public class CookieHandler {
+    public static String retrieveJSessionIdCookie(String ENVIRONMENT_HOST,
+                                                  String endpoint,
+                                                  Boolean shouldRequestBeLogged) {
+
+        Response initialRequestResponse = initiateInitialRequest(
+                ENVIRONMENT_HOST,
+                endpoint,
+                shouldRequestBeLogged
+        );
+        return extractCookieFromResponseHeader(initialRequestResponse);
+    }
+
+    private static Response initiateInitialRequest(
+            String ENVIRONMENT_HOST,
```

Comment:
This is not good coding practise, you should only use uppercase letters if they are final static variables:. This is a good guide for code styling:
https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names

Reply:
Saved, done.